### PR TITLE
Updating quay prereq doc

### PR DESCRIPTION
**Pull request by varshab1210**
**What type of PR is this?**

/kind documentation

**What does this PR do / why we need it**:

As per the screenshot in https://github.com/redhat-developer/kam/blob/master/docs/journey/day1/prerequisites/quay.md I am updating the doc

**Have you updated the necessary documentation?**

* [ ] Documentation update is required by this PR.
* [ ] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes N/A

**How to test changes / Special notes to the reviewer**: 

The first prereq in https://github.com/redhat-developer/kam/blob/master/docs/journey/day1/prerequisites/quay.md should be in sync with the screenshot

### CODE REVIEW

File: docs/journey/day1/prerequisites/quay.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Some of the Tasks in this Tutorial involve pushing images to Quay image registry.   (The image is to be built by CI Pipeline.)   Before we can start creating Kubernetes resources, we need to obtain credentials for your Quay user account.
 
- * Create `taxi` Quay repos. Login to your Quay.io account and create a repository `taxi`
+ * Create public `taxi` Quay repo. Login to your Quay.io account and create a repository `taxi`
```
Comment (varshab1210) [inline docs/journey/day1/prerequisites/quay.md:5]: @chetan-rns can you please confirm it and apply the required label.

Comment (chetan-rns) [reply docs/journey/day1/prerequisites/quay.md:5]:
```suggestion
 * Login to your Quay.io account and create a public repository `taxi`
```
Nit :)

**Review by chetan-rns — APPROVED** [review]
@varshab1210 Thanks for the PR! 
/lgtm

**Pull request merged**